libs/platform/auth/src/oauth/providers/code-grant-login.strategy.ts

```diff
@@ -16,12 +16,15 @@ export class CodeGrantAuthLoginStrategy implements AuthLoginStrategy {
     body.set('password', request.password);
 
     return this.http
-      .request<CodeGrantAuthLoginResponse>(this.config.loginUrl, {
+      .request<CodeGrantAuthLoginResponse[]>(this.config.loginUrl, {
         method: this.config.loginMethod ?? 'POST',
         credentials: 'omit',
         body,
+        headers: {
+          'Content-Type': 'application/x-www-form-urlencoded',
+        },
       })
-      .pipe(switchMap((response) => this.redirectWith(response)));
+      .pipe(switchMap((response) => this.redirectWith(response[0])));
```

Comment:
We needed it for the token but not authorize. If we want authorize to have the same format so that it's not wrapped in an array, then I will update the mock server for it as well.

Reply:
Ok let's keep it, seems like BAPI already using this format anyways.